lib/util.js

```diff
@@ -0,0 +1,25 @@
+var url = require("url");
+var qs = require("qs");
+
+/**
+ * getNextLink
+ * @function
+ * @param {Object} response A response object returned from calling 'client.makeRequest'
+ * @returns A parsed version of the link to the subsequent page, or null if no such page exists.
+ */
+var getNextLink = function (response) {
```

Comment:
Since this function really only cares about the headers we should probably just pass that in instead of the response. Makes it slightly more agnostic.